pom.xml

```diff
@@ -1304,7 +1304,7 @@
         <jdk>11</jdk>
       </activation>
       <properties>
-        <java.version>11</java.version>
+        <java.version>8</java.version>
```

Comment:
Does this set the target to 8? Is it supposed to 8, or 1.8, or does it not matter?

Reply:
oops, fixed